sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -321,6 +320,12 @@ export interface UpdateTagPropertiesOptions extends OperationOptions {
     canWrite?: boolean;
 }
 
+// @public
+export interface UploadBlobOptions extends OperationOptions {
+    // (undocumented)
+    chunkSize?: number;
```

Comment:
Per our feature crew discussion, we may end up not exposing chunk size in the operation-level API.